stable/nginx-ingress/templates/controller-daemonset.yaml

```diff
@@ -21,6 +21,7 @@ spec:
         component: "{{ .Values.controller.name }}"
         release: {{ .Release.Name }}
     spec:
+      serviceAccount: {{- if .Values.controller.serviceAccount }} {{ .Values.controller.serviceAccount }}{{- else }} default{{- end }}
```

Comment:
use `serviceAccountName` instead, `serviceAccount` is deprecated https://kubernetes.io/docs/api-reference/v1.6/#podspec-v1-core